book/06-github/sections/1-setting-up-account.asc

```diff
@@ -1,95 +1,96 @@
-=== Account Setup and Configuration
+=== Hesap Kurulumu ve Yapılandırma
 
 (((GitHub, user accounts)))
-The first thing you need to do is set up a free user account.
-Simply visit https://github.com[], choose a user name that isn't already taken, provide an email address and a password, and click the big green ``Sign up for GitHub'' button.
+İlk yapmanız gereken şey ücretsiz bir kullanıcı hesabı oluşturmaktır.
+https://github.com[] adresine gidin, henüz alınmamış bir kullanıcı adı seçin, e-posta adresinizi girin ve bir şifre belirleyin, ardından büyük yeşil "Sign up for GitHub" düğmesine tıklayın.
 
-.The GitHub sign-up form.
-image::images/signup.png[The GitHub sign-up form.]
+.GitHub kayıt formu.
+image::images/signup.png[GitHub kayıt formu.]
 
-The next thing you'll see is the pricing page for upgraded plans, but it's safe to ignore this for now.
-GitHub will send you an email to verify the address you provided.
-Go ahead and do this; it's pretty important (as we'll see later).
+Göreceğiniz bir sonraki şey, yükseltilmiş planların fiyatlandırma sayfasıdır, ancak şimdilik bunu yok sayabilirsiniz.
+GitHub, e-posta adresinizi doğrulamak için size bir e-posta gönderecektir.
+İşleme devam edin ve şunu yapın: çok önemlidir (daha sonra göreceğimiz gibi).
```

Comment:
burada do this derken bir üst cümle, yani e-postayı doğrulaması kastedilmiş. "İşleme devam edin ve şunu yapın: çok önemlidir" şeklinde yazınca sanki devamında anlatılacak şeye vurgu yapılmış gibi duruyor. burada birebir çeviri yapmak yerine daha anlam odaklı "bunu yaptığınızdan emin olun" gibi bir cümle kullanılabilir

Reply:
Doğrudan o satırı silsek nasıl olur? Biraz anlamsız duruyor sanki. Yukarıda zaten çok basit bir işlem yapıldı (isim ve şifre belirleyip yeşil düğmeye basmak).